### PR TITLE
Remove all user- and translator-facing instances of 'dapp'

### PR DESCRIPTION
**Pull request by rekmarks**
Self-explanatory.

### CODE REVIEW

**Review by whymarrh — DISMISSED** [review]
LGTM!

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -1755,7 +1755,7 @@
   },
   "decryptMessageNotice": {
     "message": "$1 would like to read this message to complete your action",
-    "description": "$1 is website or dapp name"
+    "description": "$1 is web3 site or application name"
```
Comment (whymarrh) [inline app/_locales/en/messages.json:1758]:
```suggestion
    "description": "$1 is the web3 site name"
```

File: app/_locales/en/messages.json [hunk 2]
```diff
@@ -1775,7 +1775,7 @@
   },
   "encryptionPublicKeyNotice": {
     "message": "$1 would like your public encryption key. By consenting, this site will be able to compose encrypted messages to you.",
-    "description": "$1 is website or dapp name"
+    "description": "$1 is web3 site or application name"
```
Comment (whymarrh) [inline app/_locales/en/messages.json:1778]:
```suggestion
    "description": "$1 is the web3 site name"
```

**Review by brad-decker — DISMISSED** [review]

**Review by whymarrh — CHANGES_REQUESTED** [review]

**Review by whymarrh — DISMISSED** [review]
LGTM!

**Review by Gudahtt — DISMISSED** [review]
LGTM!

But could you also lower-case the one place where `Web3` is currently capitalized?

**Comment by rekmarks** [comment]
@Gudahtt, thanks for reminding me!

**Review by whymarrh — DISMISSED** [review]
3rd time's the charm 💫

**Comment by rekmarks** [comment]
@whymarrh merge conflicts 😭

**Review by Gudahtt — APPROVED** [review]

**Pull request merged**